Command/SmokeTestRunCommand.php

```diff
@@ -341,4 +335,61 @@ public function getTestNames()
     {
         return \array_keys($this->smokeTests);
     }
+
+    /**
+     * filter exluding the tests with WIP label
+     * @param array $smokeTests
+     * @return array
+     */
+    public function filterSkipLabels($smokeTests = [], $skipLabels, &$skipped): array
+    {
+        return array_filter($smokeTests, function ($smokeTestInfo) use ($skipLabels, &$skipped) {
+            $key = $smokeTestInfo['id'];
+            if (!\in_array($skipLabels[0], $smokeTestInfo['labels'])) {
+                return $smokeTestInfo;
+            } else {
+                $skipped[$key] = $smokeTestInfo;
```

Comment:
In this case the else condition shouldn't be needed because you have the if with a return. The could could be like this
```
$key = $smokeTestInfo['id'];
if (!\in_array($skipLabels[0], $smokeTestInfo['labels'])) {
    return $smokeTestInfo;
}

$skipped[$key] = $smokeTestInfo;
```